src/main/java/uk/gov/digital/ho/pttg/dto/Individual.java

```diff
@@ -5,15 +5,24 @@
 import lombok.Getter;
 import lombok.ToString;
 
+import java.io.Serializable;
 import java.time.LocalDate;
 
 @Getter
 @AllArgsConstructor
 @EqualsAndHashCode
 @ToString
-public class Individual {
+public class Individual implements Serializable {
     private String firstName;
     private String lastName;
     private String nino;
     private LocalDate dateOfBirth;
+
+    public void setFirstName(String firstName) {
```

Comment:
Could we avoid making this object mutable by passing name changes through the retry context instead?

Reply:
Not anymore as the retry policy is dead.  We could make this immutable but it needs properly thinking about and unfortunately I am out of time.